### PR TITLE
CBL208 - Simple RNN Encode-Decoder for Translation

### PR DESCRIPTION
**Pull request by siddharthchaurasia-tudip**
- Updated the timing instruction in the notebook as needed.
- Reference buganizer: http://b/153000152

### CODE REVIEW

File: courses/machine_learning/deepdive2/text_classification/labs/rnn_encoder_decoder.ipynb [hunk 1]
```diff
@@ -1224,7 +1224,7 @@
    "source": [
```
Comment (munnm) [inline courses/machine_learning/deepdive2/text_classification/labs/rnn_encoder_decoder.ipynb:1224]: <p>I think this reads better as </p><p>"Let's now average the&nbsp;<code>bleu_1</code>&nbsp;and&nbsp;<code>bleu_4</code>&nbsp;scores for all the sentence pairs in the eval set.&nbsp;The next cell takes some time to run, the bulk of which is decoding the 6000 sentences in the validation set. Please wait unitl completes."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

File: courses/machine_learning/deepdive2/text_classification/solutions/rnn_encoder_decoder.ipynb [hunk 2]
```diff
@@ -971,7 +971,7 @@
    "cell_type": "markdown",
```
Comment (munnm) [inline courses/machine_learning/deepdive2/text_classification/solutions/rnn_encoder_decoder.ipynb:971]: <p>same as above, </p><p>I think this reads better as </p><p>"Let's now average the&nbsp;<code>bleu_1</code>&nbsp;and&nbsp;<code>bleu_4</code>&nbsp;scores for all the sentence pairs in the eval set.&nbsp;The next cell takes some time to run, the bulk of which is decoding the 6000 sentences in the validation set. Please wait unitl completes."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/875/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

File: courses/machine_learning/deepdive2/text_classification/labs/rnn_encoder_decoder.ipynb (hunk 1 above)
Comment (siddharthchaurasia-tudip) [reply courses/machine_learning/deepdive2/text_classification/labs/rnn_encoder_decoder.ipynb:1224]: Hi @munnm , 
Updated the instruction as per your suggestion.
Thanks!

File: courses/machine_learning/deepdive2/text_classification/solutions/rnn_encoder_decoder.ipynb (hunk 2 above)
Comment (siddharthchaurasia-tudip) [reply courses/machine_learning/deepdive2/text_classification/solutions/rnn_encoder_decoder.ipynb:971]: Hi @munnm ,

Updated the instruction as per you suggestion.

Thanks!

**Comment by jackykumar-tudip** [comment]
@munnm Please review the changes made by Siddharth and feel free to merge the PR (if everything is okay).

**Comment by jackykumar-tudip** [comment]
@munnm Any update here?

**Pull request merged**